File: swish.py
import urllib2
import json
from bs4 import BeautifulSoup

req = urllib2.Request('https://www.swishanalytics.com/optimus/optimus-x')
response = urllib2.urlopen(req)
page = response.read()
soup = BeautifulSoup(page)

variable = str(soup.find_all('script')[8])

data = json.dumps(variable[425:425+variable[425:].index(';')], sort_keys=True, separators=(',',':'))
dataLoad = json.loads(data)

print dataLoad

File: swish.js
var cheerio = require("cheerio")
var request = require("request")

request('https://www.swishanalytics.com/optimus/optimus-x', function (error, response, html) {
  if (!error && response.statusCode == 200) {
    var $ = cheerio.load(html)
    var variable = $('script')[8].children[0].data
   	var substr = variable.substring(variable.indexOf("this.masterPlayerArray = "))
   	var data = substr.substring(substr.indexOf("= ")+2,substr.indexOf(';'))
    console.log(data)
  }
});